client/src/app/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { Inter } from "next/font/google";
```

Comment:
Woah wtf
https://fonts.google.com/specimen/Inter
NextJS just grabs us fonts like this?? Or maybe I'm misinterpreting something

Reply:
yeah man next pre-packed some fonts for ya right out of the box lol